crates/iceberg/src/spec/table_metadata.rs

```diff
@@ -768,7 +769,7 @@ pub struct SnapshotLog {
     /// Id of the snapshot.
     pub snapshot_id: i64,
     /// Last updated timestamp
-    pub timestamp_ms: i64,
+    pub timestamp_ms: TimestampMillis,
```

Comment:
I think we should provide a method to return `TimestampMillis` rather changing the field.

Reply:
Reverted this change and added a method named `timestamp`.